test/artefacts/landlord-immigration-check/B1 1PW/yes/yes/non-eea/no/no/no/no/no/no/yes.txt

```diff
@@ -0,0 +1,14 @@
+
+
+You must [check with the Home Office](https://eforms.homeoffice.gov.uk/outreach/lcs-application.ofml) to find out if the tenant is allowed to rent property. You’ll need the person’s Home Office reference number.
+
+You’ll get an answer within 2 working days.
+
+If the person’s allowed to rent, the Home Office will tell you if you need to check their documents again later.
```

Comment:
@ikennaokpala Do you happen to know which commits are over the character limit, so I can correct them?